packages/web/src/components/spa/HackAUBG/VideoSection/video_section.css

```diff
@@ -24,20 +27,28 @@
     justify-content: center;
     align-items: center;
     display: flex;
-    padding: 3% 0;
     padding: 3% 0%;
     text-align: justify;
 }
 
 @media only screen and (max-width: 900px) {
+    .video-container {
+        height: 90vh;
+    }
+
     .video-header {
-        text-size: 0.5rem;
-        text-align: justify;
+        font-size: 1.5em;
+        text-align: center;
     }
     .video {
         width: 80%;
         justify-content: center;
         align-items: center;
         display: flex;
     }
+
+    .video iframe {
```

Comment:
Isn't this suppose to be equivalent to .video div? Since the div video is only used for holding the iframe, you can remove the div and add the classname to the iframe tag, then move all css from iframe to the .video class.